src/main/java/com/google/flourbot/logic/Executor.java

```diff
@@ -0,0 +1,42 @@
+package com.google.flourbot.executor;
+
+import com.google.flourbot.datastorage.FirebaseDataStorage;
+import com.google.flourbot.entity.Macro;
+import com.google.flourbot.entity.trigger.Trigger;
+import com.google.flourbot.entity.action.Action;
+import com.google.flourbot.entity.EntityModule;
+
+import java.util.Map;
+import java.lang.Exception;
+import com.google.cloud.firestore.QueryDocumentSnapshot;
+
+// The Logic class of the server
+public class Executor {
+
+    private final EntityModule entityModule;
+    //TODO: Add GoogleSheetHandler here when ready.
```

Comment:
please read the initializer comment about that.

Reply:
Created #11